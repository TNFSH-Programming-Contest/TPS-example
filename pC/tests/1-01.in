386109434857.273634 210698459416.234449 75122546327.491957
