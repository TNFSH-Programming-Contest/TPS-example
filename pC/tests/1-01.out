1082932715510.58349609
